pkg/instrumentation/sdk_test.go

```diff
@@ -36,6 +36,12 @@ import (
 	"github.com/open-telemetry/opentelemetry-operator/internal/config"
 )
 
+var someNamespace = corev1.Namespace{
```

Comment:
There is some internal checks to add the instance id only if the ns and other values are present.

In some ways namespaces will always be present, so in order to make the tests more consistent I've changed the empty `corev1.Namespace{}` to a valid namespace with Name.

After that the internal checks started to add the instance id and I've changed the assertitions

Reply:
changed tks! 👍